### ISSUE TITLE
Build OCIS docker images with eos support:

### ISSUE THREAD

**Issue by butonic** [owncloud/product#297]
cc @mbarz @wkloucek 

https://gitter.im/cs3org/cern-golive?at=5ff720449632f63d8701508a
```
Hugo G. Labrador @labkode Jan. 07 12:39
Is getting really painful to deploy OCIS outside the full stack
I'm trying now to deploy it as it it as only change the registry to what we need
Not ideal, but is the fastest workaround

Hugo G. Labrador @labkode Jan. 07 15:45
@butonic @refs can we have a ocis tag with eos-builtin, should be straighforward to add to your CI based on this work: cs3org/reva#1389

Jörn Friedrich Dreyer @butonic Jan. 07 15:46
@labkode do you just need a github label?

Hugo G. Labrador @labkode Jan. 07 15:52
Only a Dockerhub label
Basically, in your Drone
is a matter of duplicating this code:
https://github.com/SamuAlfageme/reva/blob/7eb0ff1859feb9d19ee69876bb3f28ba957999d0/.drone.yml#L81
Like that, for every latest or v1.0.0 tag in Dockerhub there will one named latest-eos and v1.0.0-eos
That contains the binaries inside
That allows us to use OCIS out of the box without having to rely on standalone Reva or custom OCIS build
So far authentication works out of the box and I'm dealing with the EOS part right now
```
